src/OrchardCore.Modules/OrchardCore.ArchiveLater/Migrations.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using OrchardCore.ArchiveLater.Indexes;
+using OrchardCore.ArchiveLater.Models;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.ContentManagement.Records;
+using OrchardCore.Data.Migration;
+using YesSql.Sql;
+
+namespace OrchardCore.ArchiveLater;
+
+public class Migrations : DataMigration
+{
+    private readonly IContentDefinitionManager _contentDefinitionManager;
+
+    public Migrations(IContentDefinitionManager contentDefinitionManager)
+    {
+        _contentDefinitionManager = contentDefinitionManager;
+    }
+
+    public int Create()
+    {
+        _contentDefinitionManager.AlterPartDefinition(nameof(ArchiveLaterPart), builder => builder
+            .Attachable()
+            .WithDescription("Adds the ability to schedule content items to be archived at a given future date and time."));
+
+        SchemaBuilder.CreateMapIndexTable<ArchiveLaterPartIndex>(table => table
+            .Column<string>(nameof(ArchiveLaterPartIndex.ContentItemId))
+            .Column<bool>(nameof(ArchiveLaterPartIndex.Latest))
+            .Column<bool>(nameof(ArchiveLaterPartIndex.Published))
+            .Column<DateTime>(nameof(ArchiveLaterPartIndex.ScheduledArchiveDateTimeUtc))
```

Comment:
Idem Published and Latest at the end

Reply:
Do we need to add them in MapIndexTable?